scripts/release-windows

```diff
@@ -82,3 +82,7 @@ osslsigncode -pkcs12 /tmp/heroku-windows-key \
   -n 'Heroku CLI' \
   -i https://toolbelt.heroku.com/ \
   -in $INSTALLER_BASE/installer.exe -out $INSTALLER
+
+set -x
+
+aws s3 cp --cache-control max-age=300 $INSTALLER s3://heroku-cli-assets/branches/$CHANNEL/$INSTALLER_EXE
```

Comment:
can you increase this max-age to a full day and cfreate an invalidation for the file like we do here? https://github.com/heroku/cli/blob/master/scripts/release-deb#L131